test/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_service_test.clj

```diff
@@ -398,39 +398,50 @@
             :error))))))
 
 (deftest ^:integration debouncing-test
-  (testing "Default debouncing behavior"
+  (testing "debouncing"
     (let [root-dir (fs/temp-dir "debounce")
           actual-events (atom [])
-          actual-calls (atom 0)
+          callback-invocations (atom 0)
           callback (fn [e]
                      (swap! actual-events concat e)
-                     (swap! actual-calls inc))
+                     (swap! callback-invocations inc))
           app (tk/boot-services-with-config watch-service-and-deps {})
           svc (tk-app/get-service app :FilesystemWatchService)
-          event-cadence (quot watch-core/window-min 2)]
+          event-cadence (/ watch-core/window-min 2)]
```

Comment:
😆 no, I doubt there would be.  The reason for this change was simply that `quot` "looked weird" and `/` is the usual way of doing division.

Actually, I think things would go 💥  in that case, since I think `Thread/sleep` requires an integer.